sdks/java/core/build.gradle

```diff
@@ -77,13 +90,16 @@ dependencies {
   shadow project(path: ":model:pipeline", configuration: "shadow")
   shadow project(path: ":model:fn-execution", configuration: "shadow")
   shadow project(path: ":model:job-management", configuration: "shadow")
+  shadow project(path: ":model:fn-execution", configuration: "shadow")
+  shadow project(path: ":sdks:java:transform-service:launcher") // not "shadow" configuration, which is an uberjar
```

Comment:
Why do we need the "shadow" configuration for other projects referenced above ?

Note that after https://github.com/apache/beam/pull/30190/files#diff-c85538d3151bb7ac1847f9b888558c84a82705624f6fc453648aba1fa135c6d8 this is not an uber jar anymore.

Reply:
Ah, I'll try it. We may not need the shadow configurations of the other projects either. It is confusing and changes based on whether the shadow plugin is used in a project.